### PR TITLE
Feat-672/dingtalk-notification

### PR DESCRIPTION
**Pull request by lukman7788**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Add support for dingtalk notification

## How did you implement / how did you fix it  
1. Create account on dingtalk
2. Register `robot` on dingtalk => for getting access_token
3. Add config pushover on monika.yml
4. Add `keyword` on dingtalk apps => `Monika` (case sensitive)
image
image
Select created robot
image
Add keyword
image

Test case:
1. Welcoming + Incident
image

2. Report
image

3. Recovery
image

### CODE REVIEW

**Review by nicnocquee — CHANGES_REQUESTED** [review]
Please add in documentation as well.

**Comment by lukman7788** [comment]
> Please add in documentation as well.

updated
image

**Comment by dennypradipta** [comment]
In the screenshots provided above, were there any methods to render the HTML tag? The "Tweet this status" is broken when it rendered.

**Comment by lukman7788** [comment]
> In the screenshots provided above, were there any methods to render the HTML tag? The "Tweet this status" is broken when it rendered.

updated @dennypradipta 
image

**Review by nicnocquee — APPROVED** [review]

File: src/components/notification/channel/dingtalk.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
+/**********************************************************************************
+ * MIT License                                                                    *
+ *                                                                                *
+ * Copyright (c) 2021 Hyperjump Technology                                        *
```
Comment (gsukanto) [inline src/components/notification/channel/dingtalk.ts:4]: the year should be 2022

**Review by gsukanto — APPROVED** [review]
lgtm

**Review by sapiderman — APPROVED** [review]
👍🏽 

🦥 🐨 🐼 🐻

**Pull request merged**